src/components/common/CurrencySelector.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div v-if="visible" class="currency-selector">
```

Comment:
handle outside of component with v-if?

Reply:
it's a prop

Reply:
I suggest handling this outside via a v-if. doesn't make sense to pass in a prop and then do the v-if inside